### PR TITLE
removing duplicate and incorrect entry for payment request api

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://github.com/mozilla/platform-status/issues/550

### CODE REVIEW

File: features/web-payment-request.md [hunk 1]
```diff
@@ -1,12 +0,0 @@
----
-title: Web Payment Request API
-category: apps
-bugzilla: 1318984
-firefox_status: in-development
-spec_url: https://w3c.github.io/browser-payment-api/
-chrome_ref: 5639348045217792
-ie_ref: Payment Request API
-caniuse_ref: payment-request
```
Comment (mnoorenberghe) [inline features/web-payment-request.md]: LGTM but note that `caniuse_ref: payment-request` isn't on the remaining record so it should probably be added.

**Comment by chrisdavidmills** [comment]
Good call, thanks @mnoorenberghe ! I've added that in, and will now merge.

**Comment by evilpie** [comment]
@chrisdavidmills This is still pending.

**Comment by chrisdavidmills** [comment]
Let's get this merged then. Thanks for the prod.

**Pull request merged**